Sources/MeiliSearch/Model/Dump.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+
+/**
+ `Dump` instances represent the current status of a dump in the server
+ */
+public struct Dump: Codable, Equatable {
+
+    // MARK: Properties
+
+    /// Current hash from the build.
+    public let UID: String
+
+    /// Date when the server was compiled.
+    public let status: Status
+
+    enum CodingKeys: String, CodingKey {
+        case UID = "uid"
+        case status
+    }
+
+    public enum Status: Codable, Equatable {
+
+        case processing
+        case dumpProcessFailed
+        case done
+
+        public enum CodingError: Error {
+            case unknownStatus
+        }
+
+        public init(from decoder: Decoder) throws {
+            let container = try decoder.singleValueContainer()
+            let rawStatus = try container.decode(String.self)
+            switch rawStatus {
+            case "processing":
+                self = .processing
+            case "dump_process_failed":
+                self = .dumpProcessFailed
+            case "done":
+                self = .done
+            default:
+                throw CodingError.unknownStatus
+            }
+        }
+
+        public func encode(to encoder: Encoder) throws {
+            var container = encoder.singleValueContainer()
+            switch self {
+            case .processing:
+                try container.encode("processing")
+            case .dumpProcessFailed:
+                try container.encode("dump_process_failed")
+            case .done:
+                try container.encode("done")
+            }
```

Comment:
What is this for? Why do we need an enum for that? 

This seems like a lot of code for something that is just a string
```
case processing
        case dumpProcessFailed
        case done
```